packages/mediapipe-core/lib/src/ffi_utils.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+import 'dart:typed_data';
+import 'package:ffi/ffi.dart';
+
+/// Converts a list of Dart strings into their C memory equivalent.
+///
+/// See also:
+///  * [prepareString]
+Pointer<Pointer<Char>> prepareListOfStrings(List<String> values) {
+  final ptrArray = calloc<Pointer<Char>>(values.length);
+  for (var i = 0; i < values.length; i++) {
+    ptrArray[i] = prepareString(values[i]);
+  }
+  return ptrArray;
+}
+
+/// Converts a single Dart string into its C memory equivalent.
+///
+/// See also:
+///  * [prepareListOfStrings]
+Pointer<Char> prepareString(String val) => val.toNativeUtf8().cast<Char>();
```

Comment:
Do we need both methods to be public since prepareListOfStrings calls this? Maybe `prepareString` -> `_prepareString`?